charts/apisix-dashboard/Chart.yaml

```diff
@@ -31,7 +31,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.6.0
+version: 0.7.0
```

Comment:
Please use 0.6.1

In this project, 0.7.0 will only be used when the app releases a new version

Reply:
Ok. i've set the version to 0.6.1 instead. thanks!